### DISCUSSION TITLE
Instructing bot to join meet only at the scheduled time

### DISCUSSION THREAD

**Discussion by skyshines** [TobiasPankner/Teams-Auto-Joiner#168] [General]
Is there any way to instruct bot to only join the bot at that scheduled time only
it joins whenever it sees someone started the meeting

more like check this only at the scheduled time
@TobiasPankner pls look into it
thanks for the bot

**Comment by TobiasPankner** [comment]
Sorry but this is not a feature of the bot. The only thing similar is to set a delay (Join x seconds after the meeting was started)
